content/de/blog/v2-release/index.md

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Datenanfragen.de 2.0: Nagelneue Oberfläche und Beschwerdegenerator",
+    "type": "blog",
+    "date": "2022-12-07T12:00:00+02:00",
```

Comment:
The slug should be translated accordingly.

Reply:
What do you mean?

Reply:
Use something other than `v2-release` for the slug.

Reply:
Why?

Reply:
But this is perfectly fine in both languages?